### PR TITLE
Added styling to and more information about home to project

### PR DESCRIPTION
**Pull request by nthurnau**
I added some sweet styling (via style.css) to this project and more information about Home(via index.html). I hope you enjoy it!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,5 +19,10 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```
Comment (gschnall) [inline index.html]: Please place scripts at the bottom, right before closing body tag. Thank you!

**Comment by nthurnau** [comment]
I made the changes you asked for. Anything else?

**Comment by gschnall** [comment]
@nthurnau it looks Great!!! _**Great Job!**_

**Comment by nthurnau** [comment]
Huzzah!

**Pull request merged**